packages/lesmis/src/views/DashboardView.js

```diff
@@ -7,17 +7,149 @@ import React from 'react';
 import styled from 'styled-components';
 import MuiContainer from '@material-ui/core/Container';
 import Typography from '@material-ui/core/Typography';
+import { useUrlParams } from '../utils';
+import { useVitalsData } from '../api';
+import { EntityVitalsItem } from '../components';
 
-const TemplateBody = styled.section`
-  background: #e5e5e5;
-  padding-top: 1rem;
+const Wrapper = styled.section`
+  padding: 0, 0, 0, 0;
   min-height: 50vh;
+  background: #fbfbfb;
 `;
 
-export const DashboardView = () => (
-  <TemplateBody>
-    <MuiContainer>
-      <Typography>Dashboard View</Typography>
-    </MuiContainer>
-  </TemplateBody>
-);
+const Container = styled(MuiContainer)`
+  margin-left: 0;
+  padding: 0, 0, 0, 0;
```

Comment:
```suggestion
  padding: 0;
```
padding shortcut. This will set padding to zero on all sides. You may not need to reset the padding though once you re-organise the MuiContainers